.travis.yml

```diff
@@ -18,6 +18,8 @@ matrix:
       python: 3.5
     - env: TOXENV=py35-asyncio
       python: 3.5
+    - env: TOXENV=py35-asyncio
+      python: 3.5.2
```

Comment:
What do you think about removing python35-asyncio env? They're almost the same; here we're testing lowest and highest supported Python.

Reply:
Yeah